Sources/TuistGenerator/Generator/LinkGenerator.swift

```diff
@@ -247,6 +248,17 @@ final class LinkGenerator: LinkGenerating { // swiftlint:disable:this type_body_
                 buildFile.applyCondition(condition, applicableTo: target)
                 pbxproj.add(object: buildFile)
                 embedPhase.files?.append(buildFile)
+            case let .packageProduct(product, _):
+                guard let productRef = productRefs.first(where: { $0.productName == product }) else {
+                    break
+                }
+
+                let buildFile = PBXBuildFile(
+                    product: productRef,
+                    settings: ["ATTRIBUTES": ["CodeSignOnCopy", "RemoveHeadersOnCopy"]]
+                )
+                pbxproj.add(object: buildFile)
+                embedPhase.files?.append(buildFile)
```

Comment:
In a raw project without Tuist, when does Xcode configure whether something should be embedded or not? Does it happen when the dependency is added?

Reply:
Honestly, I don't know right answer to this. I believe it's happening, when you add package library to linking phase.